captcha/__init__.py

```diff
@@ -0,0 +1,19 @@
+from django.conf import settings
```

Comment:
I would rather create a default config in apps.py for this

Reply:
I'll meet you halfway and setup default config in here as well. I want these values checked on package initialisation.

Reply:
I personally don't like proxy variables for settings. Code base usually ends up in a position where there is a need to check against django settings or the default conf depending on the logic you want/need.

Just stick to settings and if needed, set the defaults on a per module level using constants to store the defaults.